.rubocop.yml

```diff
@@ -23,6 +23,13 @@ Style/FrozenStringLiteralComment:
 RSpec/NestedGroups:
   Max: 4
 
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/features/**/*'
+
+RSpec/AnyInstance:
+  Enabled: false
```

Comment:
hehe nie ladnie tak wykluczać :D 
A tak serio - czytałaś może https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/AnyInstance ?
Przeczytaj proszę i zamiast exclude'ować - obsłuż zgodnie z sugestią Rubocopa

Reply:
dzieki! 
wlasciwie ta linijka nie potrzebna jest, bez niej testy i tak przechodza, a w kodzie jest juz:

  before do
    login_as(user)
    **allow(DueDateNotificationJob).to receive(:perform_at)**.and_return(true)
  end 
  
  ALE! zrobie sobie sciage z tego, ku pamieci